docs/azure/sdk/resource-management.md

```diff
@@ -0,0 +1,328 @@
+
+- Setting [Environment Variables](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager.Core/docs/AuthUsingEnvironmentVariables.md)
+
+More information and different authentication approaches using Azure Identity can be found in [this document](https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet).
+
+### Authenticate the Client
+
+The default option to create an authenticated client is to use `DefaultAzureCredential`. Since all management APIs go through the same endpoint, in order to interact with resources, only one top-level `ArmClient` has to be created.
+
+To authenticate to Azure and create an `ArmClient`, do the following:
+
+```C# Snippet:Readme_AuthClient
+using Azure.Identity;
+using Azure.ResourceManager.Core;
+using System;
+using System.Threading.Tasks;
+
+// Code omitted for brevity
+
+var armClient = new ArmClient(new DefaultAzureCredential());
+```
+
+Additional documentation for the `Azure.Identity.DefaultAzureCredential` class can be found in [this document](https://docs.microsoft.com/dotnet/api/azure.identity.defaultazurecredential).
+
+## Key concepts
+
+### Understanding Azure Resource Hierarchy
+
+To reduce both the number of clients needed to perform common tasks and the amount of redundant parameters that each of those clients take, we have introduced an object hierarchy in the SDK that mimics the object hierarchy in Azure. Each resource client in the SDK has methods to access the resource clients of its children that is already scoped to the proper subscription and resource group.
+
+To accomplish this, we're introducing 4 standard types for all resources in Azure:
+
+#### [Resource]Data
+
+This represents the data that makes up a given resource. Typically, this is the response data from a service call such as HTTP GET and provides details about the underlying resource. Previously, this was represented by a **Model** class.
+
+#### [Resource]Operations
+
+This represents a service client that's scoped to a particular resource. You can directly execute all operations on that client without needing to pass in scope parameters such as subscription ID or resource name.
+
+#### [Resource]Container
+
+This represents the operations you can perform on a collection of resources belonging to a specific parent resource.
+This mainly consists of List or Create operations. For most things, the parent will be a **ResourceGroup**. However, each parent / child relationship is represented this way. For example, a **Subnet** is a child of a **VirtualNetwork** and a **ResourceGroup** is a child of a **Subscription**.
+
+#### [Resource]
+
+This represents a full resource object which contains a **Data** property exposing the details as a **[Resource]Data** type.
+It also has access to all of the operations and like the **[Resource]Operations** object is already scoped
+to a specific resource in Azure.
```

Comment:
Code-fence member names such as properties, methods, and classes. This is done to avoid localization.

```suggestion
This represents a full resource object which contains a `Data` property exposing the details as a **[Resource]Data** type. It also has access to all of the operations and like the **[Resource]Operations** object is already scoped to a specific resource in Azure.
```